### PR TITLE
Updating documentation for releases with the new labels to be used to onboard components

### PR DESCRIPTION
**Pull request by avalcepina**
# Description
The release agent has been updated to use `circleci.com/component-name` and `circleci.com/version` in place of `app` and `version` for detecting new components on the cluster. Old labels are still supported for the time being

# Reasons
https://circleci.atlassian.net/browse/REL-2978

Thsi will be merged only after the new release agent version has been released.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/release/configure-your-kubernetes-components.adoc [hunk 1]
```diff
@@ -63,11 +63,16 @@ spec:
...
+        circleci.com/component-name: example-deployment
+        circleci.com/version: 1.0.0
 ----
 
-NOTE: Do not add `version` to the selector labels. These labels can not be changed later, and `version` changes frequently.
+NOTE: Do not add `circleci.com/version` and `circleci.com/component-name` to the selector labels. These labels can not be changed later, and `circleci.com//version` should change for every new release.
+In the provided example `app` is being used in the selector, but any other label can be used in its place.
+WARNING: if you were using the release agent prior to version `1.2.0` you were using `app` and `version` in place of `circleci.com/component-name` and `circleci.com/version`. While `app` and `version` are still supported, we recommend to migrate to the new labels as soon as possible.
+Support for the old labels will be dropped in one of the next releases. 
+Also be aware that, after migrating to the new labels, rolling back to versions that used the old labels will be supported only for deployments and rollouts managed through Helm.
```
Comment (rosieyohannan) [inline jekyll/_cci2/release/configure-your-kubernetes-components.adoc:75]:
```suggestion

[WARNING]
===
If you were using the release agent prior to version `1.2.0` you were using `app` and `version` in place of `circleci.com/component-name` and `circleci.com/version`. While `app` and `version` are still supported, we recommend to migrate to the new labels as soon as possible.

Support for the old labels will be dropped in a future release.
 
After migrating to the new labels, rolling back to versions that used the old labels will be supported only for deployments and rollouts managed through Helm.
```

Comment (rosieyohannan) [reply jekyll/_cci2/release/configure-your-kubernetes-components.adoc:75]: Maybe a warning like this should be added to the update guide too? https://circleci.com/docs/release/update-the-kubernetes-release-agent/

Comment (avalcepina) [reply jekyll/_cci2/release/configure-your-kubernetes-components.adoc:75]: Yes it makes sense. I will add it

Comment (avalcepina) [reply jekyll/_cci2/release/configure-your-kubernetes-components.adoc]: I added it.

File: jekyll/_cci2/release/configure-your-kubernetes-components.adoc
Comment (sds0307) [inline jekyll/_cci2/release/configure-your-kubernetes-components.adoc:1]: question: Should we also update the definition of a [Component](https://circleci.com/docs/release/releases-overview/#component)? esp where it says `...that share a common "app" label.`

Comment (alexazl) [reply jekyll/_cci2/release/configure-your-kubernetes-components.adoc:1]: Good catch @sds0307! We should update that, as well.

**Comment by alexazl** [comment]
Once we make the change that Sandeep called out, this looks good to me! 👍🏽

File: jekyll/_cci2/release/configure-your-kubernetes-components.adoc
Comment (avalcepina) [reply jekyll/_cci2/release/configure-your-kubernetes-components.adoc:1]: Done

**Review by sds0307 — APPROVED** [review]

**Pull request merged**